### PR TITLE
SPUI-1021 > Create empty state for country search

### PR DESCRIPTION
**Pull request by Anton-Tesenchuk**

### CODE REVIEW

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import '../../../notifier/kyc_countries/kyc_countries_notipod.dart';
+
+class EmptyState extends HookWidget {
+  const EmptyState({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final state = useProvider(kycCountriesNotipod);
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart:16]: remove

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:simple_kit/simple_kit.dart';
+
+import '../../../notifier/kyc_countries/kyc_countries_notipod.dart';
+
+class EmptyState extends HookWidget {
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart:9]: rename to `EmptySearchResult`

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  const EmptyState({
+    Key? key,
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart:11]: add `required String text`

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      children: <Widget>[
+        const SpaceH60(),
+        Text(
+          'No results for',
+          style: sBodyText1Style.copyWith(
+            color: colors.grey1,
+          ),
+        ),
+        Text(
+          state.countryNameSearch,
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart:30]: use `text` argument

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:simple_kit/simple_kit.dart';
+
+import '../../../notifier/kyc_countries/kyc_countries_notipod.dart';
+
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart:8]: rename file to `empty_search_result.dart` and move under `lib/app/shared/components`

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart (hunk 1 above)
Comment (Anton-Tesenchuk) [reply lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart]: done

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart (hunk 2 above)
Comment (Anton-Tesenchuk) [reply lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart]: done

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart (hunk 3 above)
Comment (Anton-Tesenchuk) [reply lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart]: done

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart (hunk 4 above)
Comment (Anton-Tesenchuk) [reply lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart]: done

File: lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart (hunk 5 above)
Comment (Anton-Tesenchuk) [reply lib/app/shared/features/kyc/view/components/empty_state/empty_state.dart]: done

**Pull request merged**